src/main/kotlin/no/nav/familie/oppdrag/service/OppdragService.kt

```diff
@@ -20,12 +20,11 @@ class OppdragService(
 
     @Transactional(rollbackFor = [Throwable::class])
     fun opprettOppdrag(utbetalingsoppdrag : Utbetalingsoppdrag, oppdrag: Oppdrag, versjon: Int) {
+        LOG.debug("Lagrer oppdrag i databasen "+oppdrag.id)
+        oppdragLagerRepository.opprettOppdrag(OppdragLager.lagFraOppdrag(utbetalingsoppdrag, oppdrag), versjon)
 
         LOG.debug("Legger oppdrag på kø "+oppdrag.id)
         oppdragSender.sendOppdrag(oppdrag)
```

Comment:
Hvorfor har du endret på denne flyten? Dette treffer ikke bare e2e. Vi ønsker ikke å lagre oppdrag i familie-oppdrag sin db før vi vet at de er lagt på kø. Synes denne endringen bør tas bort, vi kan ikke endre på den opprinnelige flyten for å få e2e til å fungere.